Likely just a port conflict (the range used in the test only covers 3 ports)


09:21:38.371 [ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.091 s <<< FAILURE! - in org.apache.flink.queryablestate.network.AbstractServerTest
09:21:38.371 [ERROR] testPortRangeSuccess(org.apache.flink.queryablestate.network.AbstractServerTest)  Time elapsed: 0.062 s  <<< FAILURE!
java.lang.AssertionError: expected:<0> but was:<1>
	at org.apache.flink.queryablestate.network.AbstractServerTest.testPortRangeSuccess(AbstractServerTest.java:125)
